frontend/src/components/Activity.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    interface Activity {
+        time: string;
+        symbol: string;
+        side: string;
+        qty: number;
+        price: number;
+        totalVal: number;
+        status: string;
+    }
+
+    export let data: Activity;
+
+    let color = "warning";
```

Comment:
Not great type checking / exhaustivity. A better way could be: 
```ts
const statusColors = {
    success: "success",
    rejected: "danger",
    pending: "warning",
};

$: color = statusColors[data.status];
```

Reply:
Will do & thanks!